css-two-column-layout/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>SCALE, SPEED, AGILITY</h1>
+          <p class="heading">We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+        </header>
+      </div>
+    </div>
+    <div class="row">
+
```

Comment:
make sure that you have `column` classes within the rows to act as the content boxes. you want to avoid having content directly inside of a row without a `column` to hold it. I would recommend `column-half` classes being set to 50% width